gcloud-java-core/pom.xml

```diff
@@ -90,4 +90,102 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-site-plugin</artifactId>
+        <version>3.4</version>
+        <configuration>
+          <skipDeploy>true</skipDeploy>
+          <reportPlugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-changelog-plugin</artifactId>
+              <version>2.3</version>
+            </plugin>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-project-info-reports-plugin</artifactId>
+              <version>2.8</version>
+              <reportSets>
+                <reportSet>
+                  <reports>
+                    <report>index</report>
+                    <report>dependency-info</report>
+                    <report>dependencies</report>
+                    <report>dependency-convergence</report>
+                    <report>project-team</report>
+                    <report>mailing-list</report>
+                    <report>cim</report>
+                    <report>issue-tracking</report>
+                    <report>license</report>
+                    <report>scm</report>
+                    <report>dependency-management</report>
+                    <report>distribution-management</report>
+                    <report>summary</report>
+                    <report>modules</report>
+                  </reports>
+                </reportSet>
+              </reportSets>
+              <configuration>
```

Comment:
Maybe I am missing something.

Is the only difference between all these copies of the plugin is the configuration/artifcatId?

If so, then instead of have a copy of the complete plugin section would not it be simpler
to override the configuration (with what ever necessary per module - even if that includes a copy/paste parts)?